Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -119,33 +120,46 @@ public struct TuistCommand: AsyncParsableCommand {
             }
         }
 
+        let outputCompletion = {
+            WarningController.shared.flush()
+            outputLogfilePath(logFilePath)
+        }
+
         do {
-            defer { WarningController.shared.flush() }
             try await executeCommand()
+            outputCompletion()
         } catch let error as FatalError {
-            WarningController.shared.flush()
             errorHandler.fatal(error: error)
+            outputCompletion()
             _exit(exitCode(for: error).rawValue)
         } catch let error as ClientError where error.underlyingError is ServerClientAuthenticationError {
-            WarningController.shared.flush()
             // swiftlint:disable:next force_cast
             ServiceContext.current?.logger?.error("\((error.underlyingError as! ServerClientAuthenticationError).description)")
+            outputCompletion()
             _exit(exitCode(for: error).rawValue)
         } catch {
-            WarningController.shared.flush()
             if let parsedError {
                 handleParseError(parsedError)
             }
+
             // Exit cleanly
             if exitCode(for: error).rawValue == 0 {
                 exit(withError: error)
             } else {
                 errorHandler.fatal(error: UnhandledError(error: error))
+                outputCompletion()
                 _exit(exitCode(for: error).rawValue)
             }
         }
     }
 
+    private static func outputLogfilePath(_ logFilePath: AbsolutePath) {
+        // TODO:
+        // Once we introduce Noora, we should merge all the "completion" messages
+        // using the Noora's completion component.
+        print("\nLogs available at \(logFilePath.pathString)")
```

Comment:
I assume these logs will be also very useful on the CI. I'd suggest two things to make the logs more easily accessible on the CI:
- Document that logs are stored in the directory determined by the `XDG_STATE_HOME`. Developers can override the value of this to be in the cwdir and upload the logs using their CI provider.
- Consider uploading logs to the Tuist server (when connected) by default. Note we'll need to research how big the logs usually turn out to be and whether we would not be slowing down the CI too much by uploading them. I'd think that if we compress the logs, the size should be relatively reasonable, though.

Reply:
Good idea. I added a documentation page explaining how to diagnose issues using the logs. 
Regarding the logs being on the server, 100%. I'd lean on collecting the logs live as the "run" is happening through a web socket connection.